client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -1230,6 +1178,34 @@ private static void setRandomIndicesOptions(Consumer<IndicesOptions> setter, Sup
         }
     }
 
+    private static void setRandomIncludeDefaults(GetIndexRequest request, Map<String, String> expectedParams) {
+        if (randomBoolean()) {
+            request.includeDefaults(true);
```

Comment:
can we also randomize the value, just to test what we do when the value is explicitly set to false? It is the default value but I would prefer to test that codepath too. Same for the other new methods.

Reply:
Done!